test/src/test/testtool/ManagedFileHandler/common.properties

```diff
@@ -30,6 +30,9 @@ manage.db.serviceName=ibis4test-ManageDatabase
 dir.init.className=nl.nn.adapterframework.testtool.FileSender
 dir.init.runAnt=true
 
+dir.moveFile.className=nl.nn.adapterframework.testtool.FileSender
+dir.moveFile.runAnt=true
```

Comment:
The changes to the scenario aren't strictly necessary but make it easier to move the same file twice thus forcing an error, for instance.